src/types/coordinate.ts

```diff
@@ -0,0 +1,4 @@
+interface Coordinate {
+    coordX: number;
+    coordY: number;
```

Comment:
Almost always we should prefer full names, so coord shouldn't be used. but full "coordinate" is redundant because we're already in the context of Coordinate interface, so just 'x'/'y' would be ok

Reply:
fixed